dummy-data/competitions.js

```diff
@@ -0,0 +1,27 @@
+const competitions = [
+  {
+    id: "ok79bctyuii",
+    kynologiqueDepartment: "Gdynia",
+    competitionName: "Gdanskie Zawody o Piertuszke",
+    date: new Date(2022, 3, 22),
+    place: "Gdańsk",
+    manager: "Jagienka Krzywobroda",
+    judge: "Alberto Makkaroni",
+    steward: "Wiesio Kapusta",
+    secretary: "Antonina Antioniuk",
+  },
+
+  {
+    id: "ft64dfgjhy8",
+    kynologiqueDepartment: "Poznan",
+    competitionName: "Zawody obedience klas 0,1,2",
+    date: new Date(2022, 5, 1),
+    place: "Poznan",
+    manager: "Zbyszko Bogdaniec",
+    judge: "Zuzzana Zezowa",
+    steward: "Zygfryd Zaduzy",
+    secretary: "Posejdon Paktofonik",
```

Comment:
Something to thing about: How to connect contest with competitors / store (how much and how?) information about the competitors withing the contest.